doc/over_react_redux_documentation.md

```diff
@@ -0,0 +1,289 @@
+`areStatesEqual`, `areOwnPropsEqual`, `areStatePropsEqual`, and `areMergedPropsEqual`.
+While the defaults are probably appropriate 99% of the time, you may wish to override them with custom
+implementations for performance or other reasons.
+- #### `forwardRef` 
+    - If `true`, the `ref` prop provided to the connected component will be return the wrapped component.
+
+> [More information about the `connect` function](https://react-redux.js.org/api/connect#connect)
+  
+## Using Multiple Stores
+An application can have multiple stores by both utilizing the `context` prop of `ReduxProvider` and setting the 
+`context` parameter on `connect`. While this is possible, it is not recommended.
+> See: <https://redux.js.org/api/store#a-note-for-flux-users>
+
+> See: <https://stackoverflow.com/questions/33619775/redux-multiple-stores-why-not>
+
+__Multiple Stores Example:__
+```dart
+Store store1 = new Store<CounterState>(counterStateReducer, initialState: new CounterState(count: 0));
+Store store2 = new Store<BigCounterState>(bigCounterStateReducer, initialState: new BigCounterState(bigCount: 100));
+
+UiFactory<CounterProps> ConnectedCounter = connect<CounterState, CounterProps>(
+      mapStateToProps: (state) => (Counter()..count = state.count)
+    )(Counter);
+    UiFactory<CounterProps> ConnectedBigCounter = connect<BigCounterState, CounterProps>(
+      mapStateToProps: (state) => (Counter()..count = state.bigCount),
+      context: bigCounterContext,
+    )(Counter);
+    react_dom.render(
+      Dom.div()(
+        (ReduxProvider()..store = store1)(
+          (ReduxProvider()
+            ..store = store2
+            ..context = bigCounterContext
+          )(
+            Dom.div()(
+              Dom.h3()('ConnectedBigCounter Store2'),
+              ConnectedBigCounter()(
+                Dom.h4()('ConnectedCounter Store1'),
+                ConnectedCounter()(),
+              ),
+            ),
+          ),
+        ),
+      ), querySelector('#content')
+    );
+```
+
+In the case that you need to have multiple stores, here are the steps to do so:
+1. Create a `Context` instance to provide to the `ReduxProvider` and the components that will be in that context.
+    ```dart
+    final fooContext = createContext();
+    ```
+1. In the `connect` function wrapping the component, pass in the context instance.
+    ```dart
+       UiFactory<BarProps> ConnectedBar = connect<BarState, BarProps>(
+          // ... mapStateToProps
+          context: fooContext,
+       )(Bar);
+    ```
+1. In a ReduxProvider, pass in your context instance.
+    ```dart
+       // ... Wrapped in a reactDom.render()
+         (ReduxProvider()..store = store1)(
+           (ReduxProvider()
+             ..store = store2
+             ..context = bigCounterContext
+           )(
+             // ... connected componentry
+           ),
+         )
+    ```
+
+## Using Redux DevTools
```

Comment:
I definitely agree that the instructions for this is basically is the repo docs, and I should have linked directly to that. So that has been added!

I also see the step by step being verbose when that single copy and paste could work, but I feel like the step by step gives valuable clarity to each independent piece of the update and grouping the segment talking about adding the actual devtools code wouldn't made a huge difference, so I'd rather focus on each part of the addition. What do you think?

Reply:
agreement!